bin/propolis-server/src/lib/vm/mod.rs

```diff
@@ -42,6 +42,7 @@ use std::{
     time::Duration,
 };
 
+use anyhow::Context as AnyhowContext;
```

Comment:
Hot tip: if you need trait methods to be in-scope but the name is colliding, you can do
```suggestion
use anyhow::Context as _;
```